torch/onnx/symbolic_opset9.py

```diff
@@ -837,7 +837,8 @@ def softmax(g, input, dim, dtype=None):
 @parse_args("v", "t", "v")
 def softplus(g, self, beta, threshold):
     if beta != 1:
-        return _unimplemented("beta", "has to be 1")
+        beta = g.op("Constant", value_t=torch.LongTensor([beta]))
```

Comment:
Can `beta` be floating number?

Reply:
nit: can remove the above parse_args, and use `_maybe_get_const` to check if beta == 1, then use original beta value as input in the below ops. By that you won't need to recreate Constant node for beta.

Reply:
`beta` can be a floating number, modified symbolic accordingly